sdsec.adoc

```diff
@@ -14,56 +63,162 @@ Debug operations affected by Sdsec:
 * Executing the Program Buffer
 * Serving abstract commands (Access Register, Access Memory)
 
-When external debug is disallowed at the current privilege level, the hart will not enter Debug Mode. Halt requests will remain pending until debug is allowed.  
-Triggers with ACTION=1 (enter Debug Mode) will not match or fire when the effective privilege level is debug-disallowed.
+[[dbgaccpriv]]
+==== Debug Access Privilege
+
+The *debug access privilege* is defined as the privilege level for state accesses via the hart, such as Abstract Commands and Program Buffer execution. With Sdext, Debug Mode operates as if it has M-mode privilege. When Sdsec is implemented, Debug Mode accesses registers and memory using the *debug access privilege*. Attempts from Debug Mode to access state that requires a privilege level above the *debug access privilege* will fail and set `abstractcs`.CMDERR to 3. The *debug access privilege* is derived as shown in <<dbgpriv>>.
+
+[[dbgpriv]]
+[options="header"]
+[cols="5*"]
+.External Debug Configuration and Privilege
+|=====================================================================================================
+| mdbgen | SDEDBGALW  | VSEDBGALW  | UEDBGALW   | Debug Access Privilege
+| 1      | Don't care | Don't care | Don't care | M-mode                
+| 0      | 1          | Don't care | Dont care  | S/HS-Mode             
+| 0      | 0          | 1          | Don't care | VS-Mode               
+| 0      | 0          | 0          | 1          | U/VU-Mode             
+| 0      | 0          | 0          | 0          | None                  
+|=====================================================================================================
 
-The subsequent subsections describe how external debug is authorized by <<mdbgctl, M-mode debug control>> and <<submdbgctl, supervisor domain debug control>>.
+==== Maximum Allowed Resume Privilege Mode
+
+With Sdsec, the maximum privilege level that can be configured in PRV and V is determined in <<maxdbgpriv>>. The fields retain legal values when the PRV and V are configured with an illegal privilege level. Illegal privilege levels include unsupported levels and any level higher than the maximum allowed debug privilege. 
+
+[[maxdbgpriv]]
+[cols="5*", options="header"]
+.Maximum Allowed Resume Privilege Mode
+|=====================================================================================================
+| mdbgen | SDEDBGALW  | VSEDBGALW  | UEDBGALW   | Maximum Debug Allowed Privilege on resume 
+| 1      | Don't care | Don't care | Don't care | M-Mode                                         
+| 0      | 1          | Don't care | Dont care  | S/HS-Mode                                  
+| 0      | 0          | 1          | Don't care | VS-Mode
+| 0      | 0          | 0          | 1          | U/VU-Mode
```

Comment:
Imagine that an external debugger is debugging a VU-mode process inside a guest VM.  Does this line mean that the external debugger can change the V bit and break out of that VM and into a U mode process that's running directly under the hypervisor?